uiowa/uiowa
# Create multiple sites and fix site creation issue.

I broke site creation in https://github.com/uiowa/uiowa/pull/4666. This PR changes `blt bis` from throwing an error if multisite overrides exist to temporarily removing them. It does that  copying the file to the tmp directory and then putting it back. 

Another option would be to go back to the warning message and then only remove existing local settings files from multisites defined in config instead of _all_ multisites.

# How to test

- Check out this branch.
- Make sure you have some multisites overridden in `blt/local.blt.yml`.
- Run `blt uiowa:multisite:create foo.com --no-db --no-commit`
- While the command runs, open `blt/local.blt.yml` and notice any multisites config is gone. Also open the tmp directory in the repo root and see the original local.blt.yml file is there. When the command finishes, the original local.blt.yml file should be copied back to the blt directory.
- Discard changes.
